controller/src/beerocks/master/db/db.cpp

```diff
@@ -64,6 +69,10 @@ sMacAddr db::client_db_entry_to_mac(const std::string &db_entry)
 
     std::replace(entry.begin(), entry.end(), '_', ':');
```

Comment:
Consider changing the input argument to not be a const, if it is possible. If not possible, just don't use reference.

Reply:
will be fixed in the next PR